From 78b9f22700ec26bf8bc9bb1fb8c2ba69296955cf Mon Sep 17 00:00:00 2001
Message-Id: <78b9f22700ec26bf8bc9bb1fb8c2ba69296955cf.1387382496.git.minovotn@redhat.com>
In-Reply-To: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
References: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
From: Nigel Croxon <ncroxon@redhat.com>
Date: Thu, 14 Nov 2013 22:52:57 +0100
Subject: [PATCH 21/46] rdma: forgot to turn off the debugging flag

RH-Author: Nigel Croxon <ncroxon@redhat.com>
Message-id: <1384469598-13137-22-git-send-email-ncroxon@redhat.com>
Patchwork-id: 55703
O-Subject: [RHEL7.0 PATCH 21/42] rdma: forgot to turn off the debugging flag
Bugzilla: 1011720
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

Bugzilla: 1011720
https://bugzilla.redhat.com/show_bug.cgi?id=1011720

>From commit ID:
commit 8cd31adc7cc0602ba0a66b7d9b50fd016ab5530a
Author: Michael R. Hines <mrhines@us.ibm.com>
Date:   Sat Aug 3 22:54:49 2013 -0400

    rdma: forgot to turn off the debugging flag

    Ooops. We forgot to turn off the flag.

    Signed-off-by: Michael R. Hines <mrhines@us.ibm.com>
    Message-id: 1375584894-9917-3-git-send-email-mrhines@linux.vnet.ibm.com
    Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
---
 migration-rdma.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 migration-rdma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/migration-rdma.c b/migration-rdma.c
index 9cf73e3..fe6118d 100644
--- a/migration-rdma.c
+++ b/migration-rdma.c
@@ -27,7 +27,7 @@
 #include <string.h>
 #include <rdma/rdma_cma.h>
 
-#define DEBUG_RDMA
+//#define DEBUG_RDMA
 //#define DEBUG_RDMA_VERBOSE
 //#define DEBUG_RDMA_REALLY_VERBOSE
 
-- 
1.7.11.7

